### PR TITLE
Task/sdk 2388/refactor dirctcall to signedcall

### PR DESCRIPTION
**Pull request by shivamsharma2710**

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -1,4 +1,4 @@
-group 'com.example.clevertap_directcall_flutter'
+group 'com.example.clevertap_signedcall_flutter'
 version '1.0-SNAPSHOT'
```
Comment (darshanclevertap) [inline android/build.gradle]: @shivamsharma2710 Version should be 1.0.0

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-# clevertap-directcall-flutter-sdk
-Flutter SDK for CleverTap Direct Call
+# clevertap-signedcall-flutter-sdk
+Flutter SDK for CleverTap Signed Call
```
Comment (darshanclevertap) [inline README.md]: @shivamsharma2710 Need a better ReadMe for this

File: example/android/app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,65 +1,65 @@
...
 
-       <!--
-       //Out of box solution of CleverTap Android SDK
-       <service android:name="com.clevertap.android.sdk.pushnotification.fcm.FcmMessageListenerService"
-           android:exported="false">
-           <intent-filter>
-               <action android:name="com.google.firebase.MESSAGING_EVENT" />
-           </intent-filter>
-       </service>-->
+        <!--
```
Comment (darshanclevertap) [inline example/android/app/src/main/AndroidManifest.xml]: @shivamsharma2710 remove this if not needed

File: README.md (hunk 2 above)
Comment (shivamsharma2710) [reply README.md]: @darshanclevertap Yes. Will work on this.

File: android/build.gradle (hunk 1 above)
Comment (shivamsharma2710) [reply android/build.gradle]: This is resolved!

File: README.md (hunk 2 above)
Comment (shivamsharma2710) [reply README.md]: @darshanclevertap This is resolved!

File: example/android/app/src/main/AndroidManifest.xml (hunk 3 above)
Comment (shivamsharma2710) [reply example/android/app/src/main/AndroidManifest.xml]: @darshanclevertap This is resolved!

**Review by darshanclevertap — APPROVED** [review]

**Pull request merged**